web/text_layer_builder.js

```diff
@@ -188,11 +188,12 @@ class TextLayerBuilder {
   }
 
   static #enableGlobalSelectionListener() {
-    if (TextLayerBuilder.#selectionChangeAbortController) {
+    if (this.#selectionChangeAbortController) {
```

Comment:
In general, I try to avoid `this` for static fields because it breaks with subclasses:

```js
class A {
  static num = 2;
  static #num = 2;

  static logPublic() {
    console.log(A.num);
  }
  static logPrivateA() {
    console.log(A.#num);
  }
  static logPrivateThis() {
    console.log(this.#num);
  }
}

class B extends A {}

B.logPublic(); // logs 2
B.logPrivateA(); // logs 2
B.logPrivateThis(); // throws
```

Reply:
Given that `TextLayerBuilder` isn't used like that, it does feel a little like we'd be "paying" in overall code-size for something we're not using.

Also, the static `#removeGlobalSelectionListener` method (just above) already uses `this` :-)

Finally, we've lots of existing code doing that same thing in static methods and I do think that we should be consistent throughout the code-base.
However, I'm not that "excited" about the code-size impact of changing all of those cases...